wolfcrypt/src/aes.c

```diff
@@ -1697,6 +1750,14 @@ static void wc_AesDecrypt(Aes* aes, const byte* inBlock, byte* outBlock)
         if (!((keylen == 16) || (keylen == 24) || (keylen == 32)))
             return BAD_FUNC_ARG;
 
+        /* Check key length */
+        #if defined(AES_MAX_KEY_SIZE)
+            const word32 max_key_len = (AES_MAX_KEY_SIZE / 8);
```

Comment:
For ANSI C (C89) we'll need brackets around this because max_key_len isn't declared at the top of function scope.